packages/opentelemetry-metrics/src/export/ConsoleMetricExporter.ts

```diff
@@ -27,18 +28,18 @@ export class ConsoleMetricExporter implements MetricExporter {
     resultCallback: (result: ExportResult) => void
   ): void {
     for (const metric of metrics) {
-      console.log(metric.descriptor);
-      console.log(metric.labels);
+      diag.info('Descriptor', metric.descriptor);
```

Comment:
ConsoleExporters (both trace and metrics) need to print to the console, that's by design. I think we must exlude this rule in these files